astroid/const.py

```diff
@@ -15,8 +15,8 @@ class Context(enum.Enum):
 
 
 # TODO Remove in 3.0 in favor of Context
-Load = Context.Load  # pylint: disable=invalid-name
-Store = Context.Store  # pylint: disable=invalid-name
-Del = Context.Del  # pylint: disable=invalid-name
+Load = Context.Load
+Store = Context.Store
+Del = Context.Del
```

Comment:
I was hoping we could fix the false negative in pylint first but I don't have much time right now. Can we separate those three removal in a single commit so we can revert it later ? (I'm on mobile so maybe it's already the case)

Reply:
This is already the case, sorry I was on mobile.

Reply:
Don't think about it. I rebased and force pushed it after your comment 😄